frontend/src/scenes/pipeline/Overview.tsx

```diff
@@ -46,6 +47,9 @@ export function Overview(): JSX.Element {
                     shown here. <Link to={urls.pipeline(PipelineTab.Destinations)}>See all.</Link>
                 </p>
                 <DestinationsTable />
+                <div className="mt-4" />
+                <h2>New destinations</h2>
+                <DestinationOptionsTable />
```

Comment:
I don't like this tbh. That page is already hyper cluttered. Can we just have a button "+ New destination" and link that over to the right page instead?

Reply:
Changed it to look like this: ![2024-11-22 at 14 34 29](https://github.com/user-attachments/assets/ef53c454-3dba-4513-88e9-5e4be6baa565)